### PR TITLE
Graphql responses generation

### PR DESCRIPTION
**Pull request by zadigus**
[Recap of our workflow](https://softozor.github.io/github_workflow.html)

# Prerequisites

- [x] You have configured the triggering of the pre-commit hook on your local repository's clone. You need to have installed the python module `pre-commit` (listed in [the required dev modules](requirements-dev.txt)) and run the command `pre-commit install` in the repository's root folder.
- [ ] You have covered your code with unit and acceptance tests
- [ ] The feature you have implemented has no tag `@wip` any more

# Changes

Activate the code that generates the desired graphql responses based on the input json fixtures.

# How to use the feature

Run
```
make fixtures.generate
```
You will see a folder `graphql/responses`. I don't know yet what we will do with those files, therefore I propose we generate them in that folder and that's it for the time being. Frontend developers will just be provided with a `docker-compose` where they have the same fixtures generation service. That way, they can use the very same fixtures too.

### CODE REVIEW

**Comment by zadigus** [comment]
ok to test

File: fixtures-generator/graphql/responses_generator.py [hunk 1]
```diff
@@ -30,15 +31,16 @@ def _to_json(self, object, output_filename):
...
+        super().__init__(fixtures_dir, os.path.join(
+            output_dir, fixtures_set, 'Consumer'), fixtures_set)
+        self.__SHOPS_FIXTURE = helpers.get_shopozor_fixture(self._INPUT_DIR)
 
     def _produce_data(self):
         return {
             'data': {
                 'shops': {
-                    'edges': [helpers.shop_node(shop_fixture['pk'], shop_fixture['fields']) for shop_fixture in [shop for shop in self.__SHOPS_FIXTURE if shop['model'] == 'shopozor.shop']]
+                    'edges': [helpers.shop_node(shop) for shop in self.__SHOPS_FIXTURE['shops']]
```
Comment (donnerc) [inline fixtures-generator/graphql/responses_generator.py]: With Hasura, we don't have this notion of "edges" and "nodes" in the graphql schema. We don't have to "follow" edges anymore. Is this intentional ?

File: fixtures-generator/graphql/responses_generator_helpers.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+import urllib.parse
+
+
+MEDIA_URL = '/media/'
+STATIC_URL = '/static/'
+PRODUCT_THUMBNAIL_SIZE = 500
+CATEGORY_THUMBNAIL_SIZE = 250
+
+
+def shop_node(shop):
```
Comment (donnerc) [inline fixtures-generator/graphql/responses_generator_helpers.py]: does this come from the old `saleor` tests ?  We don't have this notion of `nodes` and `edges` anymore in hasura.

File: fixtures-generator/graphql/responses_generator.py (hunk 1 above)
Comment (zadigus) [reply fixtures-generator/graphql/responses_generator.py]: As I said, it's code ported from the old saleor backend; that kind of details will be solved / removed when I write the api tests.

**Pull request merged**